server/src/main/resources/secrets/directory.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Just an FYI, this is changing to be a property that's passed in...so this file will no longer be necessary.

Reply:
Should that change as part of this story or just leave this to be changed when that happens?

Reply:
not this story